intelmq/bots/collectors/twitter/collector_twitter.py

```diff
@@ -0,0 +1,119 @@
+"""
+Twitter Collector Bot
+
+Extracts tweet_text from tweets and follows urls in supported_sources. 
+Url text and tweet text are sent separately.
+
+Parameters:
+
+    feed - feed name (Twitter)
+
+    tweet_count : int     
+
+    target_timelines : comma separated names of users whose timelines are followed
+
+    follow_urls : comma separated screen_names of tweeters for which it is allowed to follow urls, currently extraction only from pastebin is supported 
+
+    include_rts : bool default False (include retweets)
+
+    exclude_replies: bool default True
+
+    timelimit: int default 84400s, how far to the past do we accept tweets?
+
+    Api login data:
+        consumer_key
+        consumer_secret
+        access_token_key
+        access_token_secret
+
+To get api login data see: https://python-twitter.readthedocs.io/en/latest/getting_started.html
+
+"""
+
+
+import time
+import twitter
```

Comment:
Can you please explain why would one not want to raise ImportError? 
The tests failed because requirements are not installed. What can be done about that?

Reply:
The problem is that if we do that, the user is completely unaware why the bot does die without writing any logs.